src/itinerary/index.tsx

```diff
@@ -16,6 +16,9 @@ const StyledItinerary = styled(Itinerary)`
     position: relative;
     list-style-type: none;
   }
+  & {
+    padding-top: ${space.m};
```

Comment:
There is no padding-bottom?

Reply:
The padding-bottom is set after all itinerary leg.

Reply:
> The padding-bottom is set after each itinerary leg.

Yes this is not ideal, but it will require a bigger refacto, if we want to change that.